src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -945,6 +948,21 @@ private ConfigComparisonResult compareConfigs() {
     return comparisonResult;
   }
 
+  private void askUserForName() {
+    TextInputDialog dialog = new TextInputDialog();
+    dialog.setTitle("Enter your name");
+    dialog.setHeaderText("For auditing purposes, enter the name(s) of everyone currently "
+        + "operating this machine.");
+    dialog.setContentText("Name:");
+    Optional<String> result = dialog.showAndWait();
+
+    if (result.isPresent()) {
+      Logger.info("Name(s) of operators, as entered interactively: " + result.get());
+    } else {
+      Logger.info("Operator(s) did not enter a name.");
```

Comment:
Should we block this / throw an exception here instead?

Reply:
I prefer this, since when developing, I often hit tabulate a lot and can just hit cancel.

My reasoning is that we can't guarantee they'll enter a correct name, so why pretend to validate? They could enter `foo` instead of empty string. Just let them do empty string.

Reply:
Ok, but if that's the case, why do we force the user to provide a value in the CLI version? Shouldn't we treat this consistently?

I'm ok with allowing the user to bypass without entering a name but, if they do, we should probably log a warning (or maybe just an info) saying the user didn't provide a name.

Reply:
We already log an INFO that user didn't provide a name (`Operator(s) did not enter a name.`)

Reply:
I can update the CLI to not fail on empty string, but I fear it's more hidden there. Especially when running via gradle, the line `<===========--> 85% EXECUTING [20s]` shows up after the request for a name, so it could be hidden.

In other words, empty string in CLI is more likely to be accidental, whereas in tabulation, it's more obvious.

Happy to make changes in either direction!

Reply:
I think an argument in favor of throwing an exception / requiring this in both cases is that the user might accidentally bypass the message box with a mis-click or mis-key. I say we should go forward with it. It's really easy to just enter "asdf" when testing here anyway.

Reply:
Or aoeu, depending on keyboard layout ;)